modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -48,18 +48,31 @@
 			</description>
 		</method>
 		<method name="abs">
-			<return type="float">
+			<return type="int">
```

Comment:
The float versions should come first, because that's alphabetical order. As-is, `--doctool .` produces diffs since it swaps them.

Reply:
Done